pyproject.toml

```diff
@@ -49,7 +49,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.8,<3.9.7 || >=3.9.8,<4.0"
```

Comment:
nit: 
```suggestion
python = ">=3.8, !=3.9.7"
```

See [Multiple Version Specifiers](https://stackoverflow.com/questions/54720072/dependency-version-syntax-for-python-poetry)

Reply:
I made the change, but I needed to add <4.0 to !=3.9.7 in order for it to work.
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/a78299fe-7158-48b6-a352-4bde14fdb755">
Now the error message is easier to understand! Thank you @kevinjqliu!